src/askscript/index.ts

```diff
@@ -3,6 +3,8 @@ import { fromAskScriptAst, createElement } from '../askjsx';
 
 export type AskScriptAstSimpleValue = null | boolean | number | string;
 
+export type AskScriptCode = string;
```

Comment:
If this type is to become API now, it would be nice to add some comment explaining what it is

Reply:
There are 4 different types defined next to each other, two of them as complex as:
```
export type AskScriptAst =
  | AskScriptAstValue
  | {
      name: string;
      props: Record<string, AskScriptAst>;
      children: AskScriptAst[];
    };
```
and no single comment for them.

I find `export type AskScriptCode = string;` self-commenting thanks to a descriptive class name.